app/adapters/application.js

```diff
@@ -1,3 +0,0 @@
-import LFAdapter from 'ember-localforage-adapter/adapters/localforage';
```

Comment:
Just to be sure, if I take a look at [the example that is being referred to from the ember-cli docs](https://github.com/edgycircle/ember-pikaday/blob/master/app/components/pikaday-input.js), then we should leave it in, but I would rename it lf-adapter.js.